mock/src/transaction.rs

```diff
@@ -342,4 +346,18 @@ impl MockTransaction {
 
         self.to_owned()
     }
+
+    /// Mark the transaction as invalid.
+    pub fn invalid(&mut self) -> &mut Self {
+        self.invalid = true;
+        self
+    }
+
+    /// Force the nonce field for the MockTransaction.
+    /// Can only be used for invalid transactions.
+    pub fn force_nonce(&mut self, nonce: u64) -> &mut Self {
```

Comment:
More intuitive to use?
```suggestion
    pub fn set_nonce(&mut self, nonce: u64) -> &mut Self {
```

Reply:
I called it `force_nonce` to make it more apparent that it normally shouldn't be used because the nonce is set automatically when you need the nonce to be valid.